### PR TITLE
Add xo badges

### PR DESCRIPTION
**Pull request by tusbar**
I like having xo badges on my repos. :D

/cc @sindresorhus

### CODE REVIEW

File: libs/live-fns/xo.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  if (xoConf.space === true) {
+    xoConf.space = 2
+  }
+
+  switch (topic) {
+    case 'status':
+      return {
+        subject: 'xo',
+        status: 'enabled',
+        color: 'cyan'
```
Comment (amio) [inline libs/live-fns/xo.js]: Better use XO's theme color `5ED9C7`

File: libs/live-fns/xo.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      subject: 'xo',
+      status: 'not enabled',
+      color: 'grey'
+    }
+  }
+
+  const xoConf = {
+    semicolon: true,
+    space: false,
+
```
Comment (amio) [inline libs/live-fns/xo.js]: This line can be removed

File: libs/live-fns/xo.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      return {
+        subject: 'xo',
+        status: 'enabled',
+        color: 'cyan'
+      }
+
+    case 'indent':
+      return {
+        subject: 'indent',
+        status: xoConf.space
```
Comment (amio) [inline libs/live-fns/xo.js]: Maybe make it a standalone function?

```javascript
const getIndent = space => {
  if (space === false) return 'tab'
  if (space === true) return '2 spaces'
  if (space === 1) return '1 space'
  return `${space} spaces`
}
```

**Comment by olstenlarck** [comment]
This should be some joke.

**Comment by amio** [comment]
@olstenlarck It's simple but still fine and useful, so why not? We have a lot of dynamic licence badges, even I used one on [README](https://github.com/amio/badgen-service/commit/df271d2fdd3385f94ec4fbc2cb741a08970108e6#diff-04c6e90faac2675aa89e2176d2eec7d8R8) 😄

**Comment by amio** [comment]
@tusbar Thanks!

**Comment by styfle** [comment]
This is a good idea. Should we make a similar one for Prettier with the iconic pink? https://prettier.io

**Comment by olstenlarck** [comment]
@amio license is one thing, some (someone') style is totally another.
Then we should add for @standard and SemiStandard. And as @styfle mention - Prettier.

In any way, such badges are ridiculous idea and they are not truly "dynamic" in heart. It not make any sense to have such "dynamic" thing, because you don't change or do anything related to that in every single commit. CI badges make sense because the service can provide you with feeling what is going on.

It not make sense, it just don't. It's more like a promotion/discoverability than anything.

**Comment by amio** [comment]
I understand your point, thanks for sharing, while I thinks there's more perspective for this.

As an author, I know static badge has better performance than live badge, I thought I would use that on a license badge which usually won't change ever. But actually as a user, I prefer to use live badge. This gives me a thought: user don't (want to) think about performance, that should be author's concern. From a user's perspective, every badge __should__ serve the best performance, and every convenience count. That's why I choose live badge for license badge as a user.

I would love to support various use cases, as long as it doesn't cost too much. Some are must-have, some are good-to-have, and of course some are not-to-have. This case is a good-to-have as I explained previously, it won't bother me to "complete" this list (of linter badges), but if someone send a PR, why refuse that?

**Comment by papb** [comment]
Does adding this functionality decreases the performance of other live badges?

**Comment by olstenlarck** [comment]
> Does adding this functionality decrease the performance of other live badges?

No. Everything is separate : )

@amio yup.

**Comment by amio** [comment]
@papb I talked a lot about perf (seems causing some confusion) :P, that case (static vs live) is to show convenience is more weight than other factor (like perf etc.) from a user's perspective.

**Comment by papb** [comment]
Oh, I see. In this case, I think this PR is good too :)

**Pull request merged**